config/analytics_pii.yml

```diff
@@ -25,6 +25,7 @@ shared:
   - one_time_password
   - hmrc_bank_validation_responses
   - dqt_teacher_status
+  - teacher_id_user_info
```

Comment:
Does it need to be added to [this](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/8b35ef6b2bf2c51d394f5dc598acb1eb08069f92/app/models/claim/personal_data_scrubber.rb#L9) list as well?

Reply:
Yes, thanks!